### DISCUSSION TITLE
Why is CLI using `console.error` to produce rebuilding messages?

### DISCUSSION THREAD

**Discussion by rmoestl** [tailwindlabs/tailwindcss#8229] [General]
Hey there,

is there any reason why CLI rebuilding messages are produced with `console.error` instead of `console.log`. See https://github.com/tailwindlabs/tailwindcss/blob/122fb89e6c49a9d0d36c4fcdb57f9f2c39de7fc6/src/cli.js#L688

I was a bit surprised when I ran into https://stackoverflow.com/questions/72053925/how-to-run-a-command-after-tailwind-jit-compiler-ran-a-build-in-watch-mode because I expected `Rebuilding...` to be typical standard output.

**Comment by adamwathan** [comment]
Hey! Here: https://github.com/tailwindlabs/tailwindcss/pull/462

**Reply by rmoestl** [reply]
Thanks, Adam!
